src/feature-viewer.js

```diff
@@ -1392,6 +1497,29 @@ function createFeature(sequence, div, options) {
                           .interpolate(object.interpolation)
                          );
             },
+            bar: function (object) {
+                var transit;
+                if (animation) {
+                    transit = svgContainer.selectAll("." + object.className)
+    //                    .data(object.data)
+                        .transition()
+                        .duration(500);
+                }
+                else {
+                    transit = svgContainer.selectAll("." + object.className);
+                }
+                transit
+//                    .data(object.data)
+                    //.transition()
+                    //.duration(500)
+                    .attr("x", function (d) {
+                        return scaling(d.x - 0.4)
+                    })
+                    .attr("width", function (d) {
+                        if (scaling(d.x + 0.4) - scaling(d.x - 0.4) < 2) return 2;
```

Comment:
How does this 0.4 determined?

Reply:
Similar implementation used in Rect chart type..